apps/sft_v2/llama3_8b.yaml

```diff
@@ -14,11 +16,11 @@ comm:
 model:
   name: llama3
   flavor: 8B
-  tokenizer_path: /tmp/Llama-3.1-8B-Instruct
+  hf_assets_path: /tmp/Meta-Llama-3.1-8B-Instruct
 
 processes:
-  scheduler: local # local | mast (not supported yet)
-  hosts: 1
+  # scheduler: local # local | mast (not supported yet)
```

Comment:
does the `scheduler` api still exist? doesnt seem like it? i can also remove this